dts/bindings/phy/st,stm32u5-otghs-phy.yaml

```diff
@@ -36,3 +36,29 @@ properties:
         /* PLL1_P_CK/2 */
         clocks = <&rcc STM32_CLOCK(AHB2, 15U)>,
                  <&rcc (STM32_SRC_PLL1_P | STM32_CLOCK_DIV(2)) OTGHS_SEL(3)>;
+
+  clock-reference:
+    type: int
+    required: true
+    enum:
+      - 0x03
+      - 0x08
+      - 0x09
+      - 0x0a
+      - 0x0e
+      - 0x0b
```

Comment:
Could this benefit from some form of constants to improve DT readability?

See this discussed for another driver: https://github.com/zephyrproject-rtos/zephyr/pull/87981#discussion_r2273814744